bin/env_variables_all.txt

```diff
@@ -1,8 +1,18 @@
+API_CLIENT_BULK_SCAN_PROCESSOR_ID=bulk_scan_processor
+API_CLIENT_BULK_SCAN_PROCESSOR_SECRET=AAAAAAAAAAAAAAAA
+API_CLIENT_DATA_STORE_ID=ccd_data
+API_CLIENT_DATA_STORE_SECRET=AAAAAAAAAAAAAAAB
+API_CLIENT_XUI_WEBAPP_ID=xui_webapp
+API_CLIENT_XUI_WEBAPP_SECRET=AAAAAAAAAAAAAAAA
```

Comment:
If these are being added to support BEFTA testing calls generated from `bulk_scan_processor`, `xui_webapp`, etc then shouldn't they:- 

* using the variable naming format used [here](https://github.com/hmcts/aac-manage-case-assignment/blob/master/aca-docker/bin/env_variables_all.txt#L1-L5). **As this allows befta to switch to the correct client just using [`userTokenClientId`](https://github.com/hmcts/aac-manage-case-assignment/blob/master/src/functionalTest/resources/features/F-201%20-%20Assign%20Access%20within%20Organisation/F-201-Base.td.json#L12)**
* added to the [add-idam-clients.sh](https://github.com/hmcts/ccd-docker/blob/master/bin/add-idam-clients.sh) (see [also](https://github.com/hmcts/aac-manage-case-assignment/blob/master/aca-docker/bin/add-idam-clients.sh#L7))

Reply:
I don't think we actually need these - have removed them and rerun the `case-document-am-api` FTA tests and they still pass

Reply:
Fixed in https://github.com/hmcts/ccd-docker/pull/119/commits/889a83335d3eeeb7391dc7c2fb83bd18961410c0